device/tests/Microsoft.Azure.Devices.Client.Test/IoTHubClientDiagnosticTest.cs

```diff
@@ -0,0 +1,188 @@
+﻿namespace Microsoft.Azure.Devices.Client.Test
+{
+    using System;
```

Comment:
Please place the using statements outside the namespace (similar to the dotnet/corefx coding guidelines).

Reply:
I find that other classes put using statements inside the namespace? I am not sure which code style should I obey?
![image](https://user-images.githubusercontent.com/5545529/32037011-ab8f38c0-ba55-11e7-8b47-81e89e475ef7.png)

Reply:
You're right: we're trying to change things and it can get complicated during the transition.
Let's keep it as you currently have it for now and we'll convert the entire project with a single format check-in later.